src/requirement-types.d.ts

```diff
@@ -54,10 +49,9 @@ type RequirementFulfillmentInformation<T = Record<string, unknown>> =
     };
 
 type DecoratedCollegeOrMajorRequirement = RequirementCommon &
-  RequirementFulfillmentInformation<{ readonly courses: readonly EligibleCourses[] }>;
+  RequirementFulfillmentInformation<{ readonly courses: readonly (readonly number[])[] }>;
 
 type CourseTaken = {
-  readonly roster: string;
```

Comment:
To confirm: We don't need the `roster` field for `CourseTaken` type because we are getting courses from most latest offering right?

Reply:
yes, so the roster doesn't do anything to the requirement graph anymore, so it's better to remove it.